packages/locales/src/nl.ts

```diff
@@ -9,6 +9,7 @@ export const templates = {
   s129fac0ae1df7c8f: 'waarschuwing',
   s13fdff160ffad275: 'Wis tekst',
   s15abf4bcf7cba79f: 'items per pagina',
+  s197b12ac6e95ad9c: str`Bericht gesloten`,
```

Comment:
There should not be a `str` function here; that should only be there when there is a variable inside the text.